save-api/src/main/kotlin/com/saveourtool/save/api/SaveCloudClientEx.kt

```diff
@@ -203,7 +203,7 @@ interface SaveCloudClientEx {
      * @throws IllegalArgumentException if [file] is not a regular file.
      * @see SaveCloudClientEx.uploadFile
      */
-    suspend fun Organization.uploadFile(
+    suspend fun OrganizationDto.uploadFile(
```

Comment:
what is the change? Only this move to DTO?

Reply:
Mostly, yes, because now `/organizations/get/list` returns the list of DTOs.

Additionally, file size specified via `Content-Length-Custom` is now required when uploading a file.